autorest/azure/auth/auth.go

```diff
@@ -0,0 +1,412 @@
+package auth
+
+// Copyright 2017 Microsoft Corporation
```

Comment:
[nit] we've updated this to 2018 other places. Though honestly, I don't know what the correct behavior is in terms of adding copyright information.

Reply:
well from what I know copyright is available for 99 years or something and this presumably would update the start time form when to count. Not sure in US, but for Berne Convention adherents you dont even require a copyright notice